specification/maps/data-plane/Microsoft.Maps/DEM/preview/1.0/elevation.json

```diff
@@ -0,0 +1,614 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Maps Elevation API",
+    "version": "1.0",
+    "description": "The Azure Maps Elevation API provides an HTTP interface to query elevation data on the  surface of the Earth. Elevation data can be retrieved at specific locations by sending  lat/lon coordinates, by defining an ordered set of vertices that form a Polyline and a  number of sample points along the length of a Polyline, or by defining a bounding box  that consists of equally spaced vertices as rows and columns. The vertical datum is EPSG:3855.  This datum uses the EGM2008 geoid model applied to the WGS84 ellipsoid as its zero height  reference surface. The vertical unit is measured in meters, the spatial resolution of the  elevation data is 0.8 arc-second for global coverage (~24 meters)."
+  },
+  "host": "atlas.microsoft.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json",
+    "application/xml"
```

Comment:
I don't see any API in this file returning XML?

Reply:
Align with all other APIs Azure maps provided, xml isn't supported as return format for now, but we want to keep it in produces now for future usage.